Makefile

```diff
@@ -39,13 +43,28 @@ GOLINT            ?= golangci-lint run --timeout 5m
 GOTOOL            ?= go tool
 GOTEST            ?= gotestsum --junitfile $(TEST_OUTPUT_DIR)/$(KREPO)-unit-tests.xml --format pkgname-and-test-fails --
 
-GOPKGS             = ./cmd/... ./pkg/apis/... ./pkg/function/... ./pkg/routing/... ./pkg/sources/... ./pkg/targets/... ./pkg/transformation/...
-LDFLAGS            = -w -s
+GOPKGS             = ./cmd/... ./pkg/apis/... ./pkg/function/... ./pkg/routing/... ./pkg/flow/... ./pkg/sources/... ./pkg/targets/... ./pkg/transformation/...
+
+GOMODULE		   = github.com/triggermesh/triggermesh
+GOPKGS_SKIP_TESTS  = $(GOMODULE)/pkg/sources/reconciler/ibmmqsource \
+					 $(GOMODULE)/pkg/targets/reconciler/ibmmqtarget \
+					 $(GOMODULE)/pkg/sources/adapter/ibmmqsource/mq \
+					 $(GOMODULE)/pkg/targets/adapter/ibmmqtarget/mq \
+					 $(GOMODULE)/pkg/targets/adapter/ibmmqtarget \
+					 $(GOMODULE)/pkg/sources/adapter/ibmmqsource \
+					 $(GOMODULE)/cmd/ibmmqsource-adapter \
+					 $(GOMODULE)/cmd/ibmmqtarget-adapter
+
 LDFLAGS_STATIC     = $(LDFLAGS) -extldflags=-static
 
 HAS_GOTESTSUM     := $(shell command -v gotestsum;)
 HAS_GOLANGCI_LINT := $(shell command -v golangci-lint;)
 
+SED				  := sed -i
+ifeq ($(shell sh -c 'uname 2>/dev/null || echo Unknown'),Darwin) 
```

Comment:
Good catch although IIRC it isn't just Darwin with the wonky sed, but the BSDs.

Reply:
✔️
https://github.com/triggermesh/triggermesh/pull/337/commits/8f88d3efaf87ae86ee4c9a56c2ff1ecf22a99260#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R65-R66